### ISSUE TITLE
mp4 video player for ST

### ISSUE THREAD

**Issue by atarijookie** [atarijookie/ce-atari#188]
I would like to do a mp4 player for ST based on ffmpeg, with most of the functionality there it could be extended to stream other video sources to ST, e.g. Chocolate Doom ;) 

I know this isn't a feature that most of the people want (as they want floppy write support, SLM printer emulation, hard drive bugs to be solved), but this thing would introduce some fun into my ordinary work stack, and it's possibly something I could impress some people with :-D so once I'll have the time, mood and other resources, I might try to do it (and might fail miserably during the process). 

So this is how I can convert the vid from mp4 to raw:
```ffmpeg -i input_video.mp4 -vf scale=w=320:h=200:force_original_aspect_ratio=decrease,pad=320:200 -r 10 -vcodec rawvideo -pix_fmt gray output_video.yuv```

...it's related to #94 - mp3 player

**Comment by ggnkua** [comment]
This reminded me of a project gasman demo'd at a demoparty that would render quake on a rpi, convert the graphics to spectrum format and then send it to the spectrum for rendering. So essentially the spectrum would only show graphics and sound and send the keyboard input back to the pi!

Alas, he never released the sources for that :( I did ask him for the project in person once but that didn't do the trick either...

**Comment by atarijookie** [comment]
Yeah, the idea is the same here, others sources not needed now, let's see how it will go ;) (some time after I finish the floppy write support - #156)
